lib/tessel/wifi.js

```diff
@@ -13,26 +13,43 @@ var Tessel = require('./tessel');
 Tessel.prototype.findAvailableNetworks = function() {
   logs.info('Scanning for available networks...');
   return this.simpleExec(commands.scanWiFi())
-    .then(function wifiScanResults(resultsJSON) {
-      // Parse the response
-      var networks = '';
-      try {
-        networks = JSON.parse(resultsJSON).results;
-      } catch (err) {
-        // Throw any unfortunate errors
-        return Promise.reject(err);
+    .then(function wifiScanResults(resultsData) {
+      // Array of network dicts
+      var networks = [];
+      // Split data into individual networks
+      var networkStrings = resultsData.split('\n\n');
+      if (networkStrings.length <= 1) {
+        return networks;
```

Comment:
So, if there is only one entry then return the empty list? Is that intentional or is that part of the work around on the next line?